main.js

```diff
@@ -130,7 +131,7 @@ class Card {
         if(this.face=== UP || this.isMatch) {
             fill('#aaa');
             ellipse(this.x, this.y, this.r);
-            image(cardFaceImg, this.x - 35, this.y - 25);
+            image(this.cardFaceImg, this.x - 70, this.y - 80);
```

Comment:
I think `this` was the kicker (pardon the pun). Without the reference to `this`, cardFaceImg was just an undefined image. That is where you were getting the problem with 'img is undefined'.
Your images are not consistent sizes so you may want to trim them so they are all the same size and shape. Otherwise, getting the image to line up over the ellipse is going to be pretty tough.